src/scripts/inclusion-bot.yaml

```diff
@@ -107,6 +119,9 @@ triggers:
       - overwhelming
       - asinine
       - unbelievable
+    why: >
+      ":TRIGGER:" arose as a derogatory reference to people with mental
+      illnesses. We want to avoid stigmatizing mental illness.
```

Comment:
would losing the "want to" be stronger? "We avoid stigmatizing mental illness"

Reply:
I think the whole thing needs a run through the writing lab, to be honest. Personally I'm okay updating again later after they've had a chance to review it, but I'm also okay with waiting if that'd be better. I just don't want this to sit around forever, which is a problem Inclusion Bot PRs have a history of doing. 😬

Reply:
i think that makes sense! i definitely think it's mergable.